src/FNA3D_Driver_SDL.c

```diff
@@ -4026,7 +4026,13 @@ static void SDLGPU_DestroyDevice(FNA3D_Device *device)
 
 static uint8_t SDLGPU_PrepareWindowAttributes(uint32_t *flags)
 {
-	return SDL_GPUSupportsShaderFormats(MOJOSHADER_sdlGetShaderFormats(), NULL);
+	uint8_t result = SDL_GPUSupportsShaderFormats(MOJOSHADER_sdlGetShaderFormats(), NULL);
+	const char* error = SDL_GetError();
+	if (error)
+	{
+		FNA3D_LogWarn("SDL_GPUSupportsShaderFormats failed: %s", error);
+	}
```

Comment:
The error buffer may inherit from a previous call, so we can do the same as the other call here:

```suggestion
	uint8_t result = SDL_GPUSupportsShaderFormats(MOJOSHADER_sdlGetShaderFormats(), NULL);
	if (!result)
	{
		FNA3D_LogWarn("SDL_GPUSupportsShaderFormats failed: %s", SDL_GetError());
	}
```